code/ui/blocks/src/blocks/Stories.tsx

```diff
@@ -4,10 +4,16 @@ import { styled } from '@storybook/theming';
 import { DocsContext } from './DocsContext';
 import { DocsStory } from './DocsStory';
 import { Heading } from './Heading';
+import type { Of } from './useOf';
+import { useOf } from './useOf';
 
 interface StoriesProps {
   title?: ReactElement | string;
-  includePrimary?: boolean;
+  includePrimaryStory?: boolean;
```

Comment:
This name change is breaking, so it can't be included in a minor release. Please revert or we will need to wait until Storybook 9 to merge this PR.

Reply:
Thanks for reviewing the code! I reverted includePrimaryStories back to includePrimary.